DataFormats/L1TMuon/interface/EMTFHit.h

```diff
@@ -5,158 +5,241 @@
+    int   subsector   ; //  0 -  6.  In CSCs, 1 or 2 for ME1, 0 for ME2/3/4.  In RPCs, 1 - 6, where 1 is first chamber in EMTF sector.
+    int   subsector_RPC; // 0 -  6.  RPC sector convention (in CMSSW): subsector 3 is the first chamber in the EMTF sector.
+    int   chamber     ; //  1 - 36.  Chamber 1 starts at -5 degrees.
+    int   csc_ID      ; //  1 -  9.  For CSCs only.
+    int   csc_nID     ; //  1 - 15.  For CSCs only.  Neighbors 10 - 15, 12 not filled.
+    int   roll        ; //  1 -  3.  For RPCs only, sub-division of ring. (Range? - AWB 02.03.17)
+    int   neighbor    ; //  0 or 1.  Filled in EMTFBlockME.cc
+    int   mpc_link    ; //  1 -  3.  Filled in EMTFHit.cc from CSCCorrelatedLCTDigi
+    int   pc_sector   ; //  1 -  6.  EMTF sector that received the LCT, even those sent from neighbor sectors.
+    int   pc_station  ; //  0 -  5.  0 for ME1 subsector 1, 5 for neighbor hits.
+    int   pc_chamber  ; //  0 -  8.
+    int   pc_segment  ; //  0 -  3.
+    int   wire        ; //  0 - 111  For CSCs only.
+    int   strip       ; //  0 - 158  For CSCs only.
+    int   strip_hi    ; //  ? -  ?.  For RPCs only, highest strip in a cluster.  (Range? - AWB 02.03.17)
+    int   strip_low   ; //  ? -  ?.  For RPCs only, lowest strip in a cluster.  (Range? - AWB 02.03.17)
+    int   track_num   ; //  ? -  ?.  For CSCs only.  (Range? - AWB 02.03.17)
+    int   quality     ; //  0 - 15.  For CSCs only.
+    int   pattern     ; //  0 - 10.  For CSCs only.
+    int   bend        ; //  0 or 1.  For CSCs only.
+    int   valid       ; //  0 or 1.  For CSCs only (for now; could use to flag failing clusters? - AWB 02.03.17)
+    int   sync_err    ; //  0 or 1.  For CSCs only.
+    int   bc0         ; //  0 or 1.  Only from unpacked data? - AWB 02.03.17
+    int   bx          ; // -3 - +3.
+    int   stub_num    ; //  0 or 1.  Only from unpacked data? - AWB 02.03.17
+    int   phi_fp      ; //  0 - 4920
+    int   theta_fp    ; //  0 - 127
+    int   phzvl       ; //  0 -  6.
+    int   ph_hit      ; //  2 - 43.  (Range? - AWB 02.03.17)
+    int   zone_hit    ; //  4 - 156  (Range? - AWB 02.03.17)
+    int   zone_code   ; //  0 - 12.  (Range? - AWB 02.03.17)
+    int   fs_segment  ; //  0 - 13.  (Range? - AWB 02.03.17)
+    int   fs_zone_code; //  1 - 14.  (Range? - AWB 02.03.17)
+    int   bt_station  ; //  0 -  4.
+    int   bt_segment  ; //  0 - 25.  (Range? - AWB 02.03.17)
+    float phi_loc     ; // -20 - 60  (Range? - AWB 02.03.17)
+    float phi_glob    ; // +/-180.
+    float theta       ; // 0 - 90.
+    float eta         ; // +/-2.5.
+    float phi_sim     ; // +/-180.
+    float theta_sim   ; // 0 - 90.
+    float eta_sim     ; // +/-2.5.
+    int   is_CSC      ; //  0 or 1.
+    int   is_RPC      ; //  0 or 1.
+    int   is_GEM      ; //  0 or 1.
+    int   subsystem   ; //  1 or ?.  1 for CSC, 2 for RPC, 3 for GEM
```

Comment:
These comments are for documentation, I need to check them manually to be sure before removing question marks.  Prefer to keep them as is for now, as a "to do" reminder.